src/olympia/addons/admin.py

```diff
@@ -38,6 +39,14 @@
 log = olympia.core.logger.getLogger('z.addons.admin')
 
 
+class AddonReviewerFlagsInline(admin.TabularInline):
+    model = AddonReviewerFlags
+    # fields = ('pending_rejection', 'needs_human_review_by_mad')
```

Comment:
Does this comment serve a purpose?

Reply:
No, that was a copy/paste I need to remove - we don't need to define `fields` as we want to show all (non readonly) fields on the model anyway.

Reply:
https://github.com/mozilla/addons-server/pull/20267/commits/ea686f344a16818220baf5834366ad3e73fbf4c2